Jenkinsfile.deploy

```diff
@@ -3,11 +3,14 @@ properties([
 	disableConcurrentBuilds(),
 	disableResume(),
 	durabilityHint('PERFORMANCE_OPTIMIZED'),
+	rateLimitBuilds([
+		count: 1,
+		durationName: 'hour',
+		userBoost: true,
+	]),
 	pipelineTriggers([
-		// TODO https://github.com/docker-library/meta-scripts/issues/22
-		//upstream(threshold: 'UNSTABLE', upstreamProjects: 'meta'),
-		cron('H H/2 * * *'),
-		// (we've dropped to only running this periodically to avoid it clogging the whole queue for a no-op, which also gives build+meta more time to cycle and get deps so they have a higher chance to all go out at once -- see the above linked issue)
+		upstream('meta'),
+		cron('H H/6 * * *'), // run every few hours whether we "need" it or not
```

Comment:
This effectively sets us to run `deploy` every time `meta` succeeds, but at most once per hour (unless we run manually).  I don't personally think this is _right_, but it's maybe close enough for first pass (triggers relatively frequently, but not so frequently that we cause problems for anyone else).

I'd also add the reminder that this job running is only a blocker for _users_ to see new builds, not for dependent/child builds, so it's pretty reasonable for them to have a 1-2 hour delay (and we can always run manually if we know the builds are done and need to get a deploy out faster than that).